notesapi/v1/serializers.py

```diff
@@ -0,0 +1,74 @@
+"""
+Serializers for Notes API.
+"""
+
+import json
+from rest_framework import serializers
+from notesapi.v1.models import Note
+
+
+class NoteSerializer(serializers.ModelSerializer):
+    """
+    Student Notes Model Serializer.
+    """
+    class Meta(object):
+        """
+        Model Serializer Meta Class.
+        """
+        model = Note
+        exclude = ('user_id',)
```

Comment:
It nicer to have a `fields` attribute rather than an `exclude` attribute because it makes it clear what will be added.